docs/jwst/stpipe/config_asdf.rst

```diff
@@ -157,19 +157,21 @@ Pipeline Configuration
 ~~~~~~~~~~~~~~~~~~~~~~
 
 Pipelines are essentially steps that refer to sub-steps. As in the original cfg
-format, parameters for sub-steps can also be specified. Sub-step parameters are
-specified by using the sub-step name as the key, then underneath and indented,
-the parameters to change for that sub-step. For example, to define the
-``weight_power`` of the ``cube_build`` step in ``Spec2Pipeline`` configuration
-file, the parameter block would look as follows:
+format, parameters for sub-steps can also be specified. All sub-step parameters
+appear in a key calles `steps`. Sub-step parameters are specified by using the
+sub-step name as the key, then underneath and indented, the parameters to change
+for that sub-step. For example, to define the ``weight_power`` of the
+``cube_build`` step in ``Spec2Pipeline`` configuration file, the parameter block
```

Comment:
Need an article ("a" or "the") between "step in" and "Spec2Pipeline configuration file"